catalog/spatial/csw/spatial-csw-endpoint/src/test/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/reader/TestTransactionMessageBodyReader.java

```diff
@@ -312,7 +312,7 @@ public void testIsReadable() throws Exception {
   @Test
   public void testReadInsertFrom() throws Exception {
     Converter mockConverter = mock(Converter.class);
-    when(mockConverter.canConvert(any(Metacard.class.getClass()))).thenReturn(true);
+    when(mockConverter.canConvert(any(Class.class))).thenReturn(true);
```

Comment:
is this different that just using `any()` without any arguments?

Reply:
Not especially, but I figured I'd leave it explicit.